js/build.js

```diff
@@ -311,6 +311,10 @@
         $(this).find('.chart-styles').remove();
       }
 
+      Fliplet.Hooks.on('appearanceChanged', () => {
```

Comment:
no arrow functions (they are not supported on IE11

Reply:
Yeah, sorry 👍 Copied from another component!